src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -403,6 +404,24 @@ public SnapshotAccessRequestResponse approveRequest(UUID id) {
     return snapshotRequestDao.getById(id).toApiResponse();
   }
 
+  public SnapshotAccessRequestMembersResponse getGroupMembers(UUID id) {
+    SnapshotAccessRequestModel model = snapshotRequestDao.getById(id);
+    return new SnapshotAccessRequestMembersResponse()
+        .members(iamService.getGroupPolicyEmails(model.samGroupName(), "member"));
```

Comment:
Instead of `"member"` this should use a constant. I see we have `IamRole.MEMBER`, could that work here?